projects/plugins/jetpack/extensions/blocks/subscriptions/subscriptions.php

```diff
@@ -61,22 +68,6 @@ function register_block() {
 		);
 	}
 
-	/*
-	 * If the Subscriptions module is not active,
-	 * do not make any further changes on the site.
-	 */
-	if ( ! Jetpack::is_module_active( 'subscriptions' ) ) {
-		return;
-	}
-
-	/**
-	 * Do not proceed if the newsletter feature is not enabled
-	 * or if the 'Jetpack_Memberships' class does not exists.
-	 */
-	if ( ! class_exists( '\Jetpack_Memberships' ) ) {
```

Comment:
It was a bit weird to see this class check here since we just import the file a few lines above:

https://github.com/Automattic/jetpack/blob/adb410b4b4bd9c8298f726b2d79889a70388177a/projects/plugins/jetpack/extensions/blocks/subscriptions/subscriptions.php#L46

So I removed the check as a redundant.